Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+jupyter = "*"
+numpy = "*"
+pandas = "*"
+seaborn = "*"
+rise = "*"
```

Comment:
Don't use such versioning in production environments. Try to give precise version numbers, cause in some time any breaking change in the dependencies will be fetched automatically and kill your production deployments (or local testing while in prod all the things will be okay).